src/main/java/org/apache/spark/sql/mlsql/sources/MLSQLBinLogDataSource.scala

```diff
@@ -91,7 +91,7 @@ class MLSQLBinLogDataSource extends StreamSourceProvider with DataSourceRegister
         toPath
 
       val lastFile = HDFSOperator.listFiles(offsetPath)
-        .filterNot(_.getPath.getName.endsWith(".tmp.crc"))
+        .filterNot(f => f.getPath.getName.endsWith(".tmp.crc") || f.getPath.getName.endsWith(".tmp"))
```

Comment:
Maby you can add some comments in this line to tell people why we need to filter `.tmp ` file.